### DISCUSSION TITLE
Im tired: Rabbitmq authentication fails (no special chars in passwd in yaml)

### DISCUSSION THREAD

**Discussion by LordAragornsLaptop** [postalserver/postal#2644] [Help with using Postal] [answered]
500 error on the web when sending a test mail. And Authentication fail error in "postal logs web" for rabbitmq. Additionally post initialise echoed "postal db already exists" 

(Btw u guys should update the script where it says docker-compose. That is deprecated with docker compose v2 and alias dont work. It should be "docker compose".)

Environment: tested on debian 11 and 12.

**Comment by willpower232** [comment]
You mentioned previously that you had a couple of Postal servers, did you point them both at the same database server? That would explain the inability to initialise again.

Unfortunately only you can resolve the RabbitMQ authentication, perhaps you need to specify a different user or set up the RabbitMQ again although if you are running two Postal servers, would that break the first one?

The docker compose thing has been noted in a few places already so they'll figure it out eventually

**Reply by LordAragornsLaptop** [reply]
Oh no, I was not clear: I tested it one two servers, one was debian 11 the other 12. On each I did this: 
Run docker containers from the "prerequisites page" then run postal bootstrap and then postal initialise. After this I am able to login and create server and send a test mail, which is where where things just stop working magically. In the logs it always says the rabbitmq is not authenticating. It all makes no sense to me. And then rabbitmq workers are never able to start again, as seen with "postal status"

**Reply by willpower232** [reply]
You can try https://github.com/postalserver/install/pull/3/files to keep the containers restarting which they don't do by default.

Are you hosting the database and rabbit on the same server as Postal?

**Reply by LordAragornsLaptop** [reply]
Did already and everything on the same server. Running the mariadb and rabbitmq container then postal initialise. There must be a bug. It just doesnt make sense.  
I was really happy to transition to postal but it just doesn't work.

**Reply by willpower232** [reply]
Are you somehow preserving your previous attempts with docker volumes or something? It is a bit weird they're missing from https://docs.postalserver.io/install/prerequisites

Something must be wrong somewhere in the setup, are there any clues in the rabbit logs?

**Reply by LordAragornsLaptop** [reply, marked as answer]
Yes there is smth wrong. But this is how to solve: first run postal initialise and then the containers for mariadb and rabbitmq. Yes that shit makes no sense, but its how you solve it XD

So better update the scripts or the docs!!!
